src/sidebar/markdown-commands.js

```diff
@@ -173,6 +173,10 @@ export function toggleSpanStyle(state, prefix, suffix, placeholder) {
   return newState;
 }
 
+/**
+ * @param {string} str
+ * @param {number} pos
+ */
```

Comment:
Though it's beyond strict typing, it'd be nice to have a bit of doc about what these params, and these functions, do.

Reply:
I've added short descriptions to these two helpers.